src/components/Teams/TeamMembersPopup.jsx

```diff
@@ -1,27 +1,44 @@
 import React, { useState, useEffect, useCallback } from 'react';
-import { Button, Modal, ModalHeader, ModalBody, ModalFooter, Container, Alert } from 'reactstrap';
+import {
+  Button,
+  Modal,
+  ModalHeader,
+  ModalBody,
+  ModalFooter,
+  Container,
+  Alert,
+  TabContent,
```

Comment:
Don't see `TabContent` being used anywhere in this component, should it be removed?